<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>treesPage</class>
 <widget class="QWidget" name="treesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Деревья (ui)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="2">
    <widget class="QSpinBox" name="edtActionTypeTreeExpand">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="edtActionTemplateTreeExpand">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cmbActionTypeTreeExpand">
     <item>
      <property name="text">
       <string>Полностью свернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Полностью развернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Развернуто на уровень</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>128</width>
       <height>217</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmbActionTemplateTreeExpand">
     <item>
      <property name="text">
       <string>Полностью свернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Полностью развернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Развернуто на уровень</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblActionTemplateTreeExpand">
     <property name="text">
      <string>Шаблоны действий</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTreeProbes">
     <property name="text">
      <string>&amp;Пробы</string>
     </property>
     <property name="buddy">
      <cstring>cmbProbesTreeExpand</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="edtProbesTreeExpand">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbProbesTreeExpand">
     <item>
      <property name="text">
       <string>Полностью свернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Полностью развернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Развернуто на уровень</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_17">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="edtTreeOrgStructureExpand">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cmbTreeOrgStructureExpand">
     <item>
      <property name="text">
       <string>Полностью свернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Полностью развернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Развернуто на уровень</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_16">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblTreeOrgStructure">
     <property name="text">
      <string>&amp;Структура организации</string>
     </property>
     <property name="buddy">
      <cstring>cmbTreeOrgStructureExpand</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblActionType">
     <property name="text">
      <string>&amp;Типы действий</string>
     </property>
     <property name="buddy">
      <cstring>cmbActionTypeTreeExpand</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblTreeContracts">
     <property name="text">
      <string>Договоры</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cmbContractsTreeExpand">
     <item>
      <property name="text">
       <string>Полностью свернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Полностью развернуто</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Развернуто на уровень</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="edtContractsTreeExpand">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_18">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_19">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <spacer name="horizontalSpacer_20">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbTreeOrgStructureExpand</tabstop>
  <tabstop>edtTreeOrgStructureExpand</tabstop>
  <tabstop>cmbActionTypeTreeExpand</tabstop>
  <tabstop>edtActionTypeTreeExpand</tabstop>
  <tabstop>cmbProbesTreeExpand</tabstop>
  <tabstop>edtProbesTreeExpand</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
